store/upgrade.go

```diff
@@ -0,0 +1,53 @@
+package store
+
+import "golang.org/x/exp/slices"
+
+// StoreUpgrades defines a series of transformations to apply the RootStore upon
+// loading a version.
+type StoreUpgrades struct {
+	Added   []string      `json:"added"`
+	Renamed []StoreRename `json:"renamed"`
+	Deleted []string      `json:"deleted"`
```

Comment:
If they are transformations to apply, the nomenclature in past tense doesn't make sense, how about in future/present tense as
```go
    Add []string `json:"add"`
    Rename []sStoreRename
    Delete []string
```
as you are giving instructions for what to operate on. The past tense is more appropriate for already applied operations.
```

Reply:
These are all from store v1 -- I can tweak them.